config/vlan.py

```diff
@@ -46,6 +46,13 @@ def del_vlan(db, vid):
     if clicommon.check_if_vlanid_exist(db.cfgdb, vlan) == False:
         ctx.fail("{} does not exist".format(vlan))
 
+    intf_table = db.cfgdb.get_table('VLAN_INTERFACE')
```

Comment:
One concern is , by removing it in background, user may not be aware that ip address association is removed. Also in the current Sonic image, you can add an ip address to a vlan interface for which vlan is non-existent. 

For e.g: `config interface ip add Vlan200 2.3.4.6/24` <== This would be accepted, even if Vlan200 doesn't exist.

My suggestion would be to log warning to user that Vlan still have IP address and let user remove the ip. What do you think?

Reply:
I made an update as per your suggestion to warn the user, but it requires some changes in the tests.
Also is this the correct behavior of adding IP to non-existent entries? There is an issue on same problem with my PR.(https://github.com/Azure/sonic-buildimage/issues/6358).